net/netdev/netdev_register.c

```diff
@@ -313,6 +314,11 @@ int netdev_register(FAR struct net_driver_s *dev, enum net_lltype_e lltype)
             devfmt   = NETDEV_TUN_FORMAT;
             break;
 #endif
+        case NET_LL_MBIM:
+          llhdrlen = 0;
+          pktsize  = 1200;
+          devfmt   = NETDEV_MBIM_FORMAT;
+          break;
```

Comment:
That code sequence must be enclosed in conditional compilation base on CONFIG_USBHOST_CDCMBIM as are all other MAC layers.